src/main/java/com/greenfoxacademy/goddesstribesbackend/security/WebSecurityConfig.java

```diff
@@ -16,7 +15,7 @@
 
   @Autowired
   private RestAuthenticationEntryPoint restAuthenticationEntryPoint;
-//
+
```

Comment:
The problem with these changes in specific tickets that they may cause conflicts on merging, because the others might have had these lines also touched. It's the best if we stick with the story, easier to review, easier to merge, and create separate ticket for code stylistics, and get them down quickly + notify peers about them.